client/asset/btc/btc.go

```diff
@@ -2184,12 +2184,13 @@ func (btc *baseWallet) SignMessage(coin asset.Coin, msg dex.Bytes) (pubkeys, sig
 		return nil, nil, err
 	}
 	pk := privKey.PubKey()
-	sig, err := privKey.Sign(msg)
+	hash := sha256.Sum256(msg) // legacy servers will not accept this signature!
```

Comment:
btc test will use `chainhash.HashB`, just confirming you meant to use `sha256.Sum256` It is the same though https://github.com/btcsuite/btcd/blob/bf64c8bdbbbfc6db51312769742e1210b8ac8ed0/chaincfg/chainhash/hashfuncs.go#L10-L14